runtime/rococo/src/lib.rs

```diff
@@ -856,19 +856,19 @@ impl pallet_parachain_staking::Config for Runtime {
 	type Currency = Balances;
 	type MonetaryGovernanceOrigin = EnsureRootOrAllCouncil;
 	/// Minimum round length is 2 minutes (10 * 12 second block times)
-	type MinBlocksPerRound = ConstU32<{ 2 * MINUTES }>;
+	type MinBlocksPerRound = ConstU32<{ prod_or_fast!(2 * MINUTES, 2) }>;
```

Comment:
Can it be too fast to test?
Not saying it's NOK, just a reminder :D

Reply:
To be honest I'm not sure which amount of time blocks is sufficient to wait. I made it to wait 2 blocks instead of 10 as you recommended to make it faster. 😊
What time you do you think should be reasonable for this case?

Reply:
I can't tell before seeing the real tests :D If you don't find a problem with 2 blocks, just go for it